### PR TITLE
Port project baseuser modules

### PR DESCRIPTION
**Pull request by dreispt**

### CODE REVIEW

File: project_baseuser/security/project_security.xml [hunk 1]
```diff
@@ -9,7 +9,7 @@
         <!-- Project Managers (modified): no longer see all projects: will have same visibility rules as Project Users -->
         <record model="ir.rule" id="project.project_project_manager_rule">
             <field name="name">Project: project manager: does not see all (modified)</field>
-            <field name="domain_force">[]</field>
+            <field name="domain_force">[(1, '=', 0)]</field>
```
Comment (pedrobaeza) [inline project_baseuser/security/project_security.xml:12]: Why this?

**Comment by dreispt** [comment]
I have this change locally for some time; I believe that at some point, the empty `[]` brought errors or problems...

**Comment by pedrobaeza** [comment]
But this way, the rule is never applied. Is that your intention?

**Comment by dreispt** [comment]
Yes. Standard behaviour is Project Managers see all Tasks & Issues.
This "disables" that rule, and they get the same rule as other Project Users, inherited from that role.

**Comment by pedrobaeza** [comment]
OK, got it

:+1:

**Pull request merged**